include/seastar/core/temporary_buffer.hh

```diff
@@ -147,6 +149,11 @@ public:
     bool empty() const noexcept { return !size(); }
     /// Checks whether the buffer is not empty.
     explicit operator bool() const noexcept { return size(); }
+
+    friend std::ostream& operator<<(std::ostream& os, const temporary_buffer& b) {
+        return os << std::string_view{b.get(), b.size()};
```

Comment:
temporary buffer typically contains binary data, I don't feel comfortable adding a way to dump raw bytes into a terminal

Reply:
@xemul hi Pavel, yeah. i suspected this could be controversial despite that it was only used in test. but indeed, it does not make sense to dump binary to a terminal. dropped the commit adding `operator<<()`.